frontend/public/components/app.jsx

```diff
@@ -58,7 +58,8 @@ import '@patternfly/quickstarts/dist/quickstarts.min.css';
 // load dark theme here as MiniCssExtractPlugin ignores load order of sass and dark theme must load after all other css
 import '@patternfly/patternfly/patternfly-charts-theme-dark.css';
 
-const breakpointMD = 1200;
+const PF_BREAKPOINT_MD = 768;
+const PF_BREAKPOINT_XL = 1200;
 const NOTIFICATION_DRAWER_BREAKPOINT = 1800;
```

Comment:
Are these values, or comparable values, available from PF of PF React packages?

Reply:
Good thought!  It doesn't appear so:
* https://github.com/search?q=repo%3Apatternfly%2Fpatternfly-react%201200&type=code
* https://github.com/search?q=repo%3Apatternfly%2Fpatternfly-react+768&type=code

Reply:
@nicolethoen do you know if the page breakpoints that pf advises are available for export from the pf packages?

Reply:
I believe you can import CSS variable names and values via @patternfly/react-tokens.
For example
`import globalBreakpointLg from '@patternfly/react-tokens/dist/esm/global_breakpoint_lg';`
and in that case, `globalBreakpointLg` contains three piece of information
```
name: "--pf-global--breakpoint--lg"
value: "992px"
var: "var(--pf-global--breakpoint--lg)"
```

Reply:
Thanks, @nicolethoen.  Unfortunately I need only a number (e.g., `1200`) and not a number with a unit (e.g., `1200px`), so I'll have to stick with what I've got.